### PR TITLE
Profile and UserPasswordForm translation hooks

### PR DESCRIPTION
**Pull request by Jamil-Bailony**
- [x] add translation hooks to Profile page container

- [x] add translation hooks to UserPasswordForm

### CODE REVIEW

File: src/containers/ProfilePage/index.jsx [hunk 1]
```diff
@@ -109,7 +111,7 @@ export default function ProfilePage() {
               <ProfileSection
                 key={section.label}
                 eventKey={index.toString()}
-                label={section.label}
+                label={t(`${section.label}`)}
```
Comment (louisrli) [inline src/containers/ProfilePage/index.jsx:114]: fix

Comment (louisrli) [reply src/containers/ProfilePage/index.jsx:114]: never mind, i see, you're substituting the translation key here i guess

**Review by louisrli — APPROVED** [review]

File: src/containers/ProfilePage/index.jsx (hunk 1 above)
Comment (Jamil-Bailony) [reply src/containers/ProfilePage/index.jsx:114]: yeah, that's right

**Comment by Jamil-Bailony** [comment]
@Ammar-64

**Review by jennetturkkan — APPROVED** [review]

**Pull request merged**